operator/validator/event_handler.go

```diff
@@ -184,19 +193,13 @@ func (c *controller) handleValidatorAddedEvent(
 
 	// verify sig
 	if err = verifySignature(event.Signature, event.Owner, event.PublicKey, nonce); err != nil {
+		malformed = true
 		return nil, &abiparser.MalformedEventError{
 			Err: errors.Wrap(err, "failed to verify signature"),
 		}
 	}
 
 	pubKey := hex.EncodeToString(event.PublicKey)
-	// TODO: check if need
-	if ongoingSync {
-		if _, ok := c.validatorsMap.GetValidator(pubKey); ok {
-			logger.Debug("validator was loaded already")
-			return nil, nil
-		}
-	}
```

Comment:
I would say it's critical to remove it since we have to test it first

Reply:
ok we can leave it then